server/rpc.go

```diff
@@ -180,20 +180,20 @@ func verifyMsgSig(addr ethcommon.Address, msg string, sig []byte) bool {
 	return eth.VerifySig(addr, crypto.Keccak256([]byte(msg)), sig)
 }
 
-func verifyTranscoderReq(orch Orchestrator, req *TranscoderRequest, job *lpTypes.Job) bool {
+func verifyTranscoderReq(orch Orchestrator, req *TranscoderRequest, job *lpTypes.Job) (bool, error) {
```

Comment:
We could probably simplify to just return error here? Also, the RPC tests probably need to be updated to match the new signature.